tests/unit/test_compression.py

```diff
@@ -51,26 +50,6 @@ def test_compression_equal_regardless_parallel_mode():
     assert serial == parallel
 
 
-def test_compress_decompress_no_parallel():
```

Comment:
here are the tests that still use it:

https://github.com/manahl/arctic/blob/master/tests/unit/test_compress.py

https://github.com/manahl/arctic/blob/master/tests/integration/test_compress_integration.py

I think there is value in keeping these somewhere, but we could move them outside the 'test' directory if desired.